client/verta/verta/operations/monitoring/profilers.py

```diff
@@ -0,0 +1,249 @@
+# -*- coding: utf-8 -*-
+
+from collections import namedtuple
+import time
+
+from verta._tracking import entity, _Context
+from verta._protos.public.monitoring import (
+    DataMonitoringService_pb2 as _DataMonitoringService,
+)
+from verta._protos.public.monitoring.DataMonitoringService_pb2 import (
+    GetProfilerRequest,
+    CreateProfilerRequest,
+    ListProfilersRequest,
+    UpdateProfilerRequest,
+    DeleteProfilerRequest,
+    EnableProfilerRequest,
+    GetProfilerStatusRequest,
+    DisableProfilerRequest,
+    KeyValue,
+    ValueTypeEnum,
+    DeployStatusEnum as DeployStatus,
+    BuildStatusEnum as BuildStatus,
```

Comment:
I feel like removing the `Enum` suffix kind of obfuscates the usefulness of the naming convention. Would prefer keeping it.